src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -60,9 +62,17 @@ void SendLocationRequest::Run() {
     return;
   }
 
-  const smart_objects::SmartObject& msg_params = (*message_)[strings::msg_params];
-  std::list<Common_TextFieldName::eType> fields_to_check;
+  smart_objects::SmartObject& msg_params = (*message_)[strings::msg_params];
+  if (msg_params.keyExists(strings::delivery_mode)) {
+    if (parameters_permissions().allowed_params.end() ==
+        std::find(parameters_permissions().allowed_params.begin(),
+                  parameters_permissions().allowed_params.end(),
```

Comment:
@anosach-luxoft , if we do some action more than one time, we can move result of action to temporary link.